core/config/value.go

```diff
@@ -443,7 +444,7 @@ func (cv Value) getValueStruct(key string, target interface{}) (interface{}, err
 			if v2 := global.GetValue(childKey); v2.HasValue() {
 
 				if err := v2.PopulateStruct(newTarget.Interface()); err != nil {
-					return nil, err
+					return nil, errors.Wrap(err, "unable to populate struct of object target")
```

Comment:
errors are never capitalized. they're meant to he interpolated into log messages